### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by thesudhanshupanda**
image

### CODE REVIEW

File: docs/Cards/card.html [hunk 1]
```diff
@@ -0,0 +1,328 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Merakii UI | Cards </title>
+    <link rel="stylesheet" href="/CSS/style.css">
+</head>
```
Comment (shuntzhere) [inline docs/Cards/card.html:9]: Use lowercase folders to avoid confusion.

File: docs/Cards/card.html [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+                    </div>
+
+                    <div class="component-heading">Card with Shadow and Overlay Text</div>
+                    <div class="component-subHeading">   
+                    </div>
+                    <div class="example-live card-container">  
+                        <!-- Card with shadow -->         
+                          <div class="card card-shadow">
+                          <img src="https://picsum.photos/seed/picsum/300/200" alt="Card-Image">
+                          <div class="card-header">
```
Comment (shuntzhere) [inline docs/Cards/card.html:151]: Add some fixed image.

File: docs/Cards/card.html [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+                          <div class="card-info">
+                            <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sit, deserunt.</p>
+                          </div>
+                          <div class="card-footer">
+                            <a href="" class="text-btn">Read</a>
+                            <a href="" class="text-btn">Bookmark</a>
+                            <a href=""><i class="card-icon fas fa-heart"></i></a>
+                            <a href=""><i class="card-icon fas fa-share-alt"></i></a>
+                            <a href=""><i class="card-icon fas fa-ellipsis-v"></i></a>                  
+                          </div>
```
Comment (shuntzhere) [inline docs/Cards/card.html:166]: Don't leave blank href's. In case there are no files to direct use #

File: docs/Cards/card.html [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+                          </div>
+                          <!-- Card with Overlay Text -->
+                          <div class="card">
+                            <img src="https://picsum.photos/seed/picsum/300/200" alt="Card-Image">
```
Comment (shuntzhere) [inline docs/Cards/card.html:170]: Give a fixed image.

File: docs/Cards/card.html [hunk 5]
```diff
@@ -0,0 +1,328 @@
...
+                             
+                              <div class="card-info">
+                                <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sit, deserunt.</p>
+                              </div>
+                              <div class="card-footer">
+                                <a href="" class="text-btn">Read</a>
+                                <a href="" class="text-btn">Bookmark</a>
+                                <a href=""><i class="card-icon fas fa-heart"></i></a>
+                                <a href=""><i class="card-icon fas fa-share-alt"></i></a>
+                                <a href=""><i class="card-icon fas fa-ellipsis-v"></i></a>
```
Comment (shuntzhere) [inline docs/Cards/card.html:187]: Check these href's.

File: docs/Cards/card.html [hunk 6]
```diff
@@ -0,0 +1,328 @@
...
+                          </div>                                
+                      </div>
+
+                      <div class="example-code">
+                        <pre>
+                          <code class="html hljs xml">
+                            <!-- For Card with shadow add the below code -->
+                            <div class="card card-shadow">
+                              <img src="" alt="Card-Image">
+                              <!-- For overlay Text add card-title-overlay class next to card-header-->
```
Comment (shuntzhere) [inline docs/Cards/card.html:199]: Check this src

File: docs/Cards/card.html [hunk 7]
```diff
@@ -0,0 +1,328 @@
...
+                              <div class="card-info">
+                                <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sit, deserunt.</p>
+                              </div>
+                              <div class="card-footer">
+                                <a href="" class="text-btn">Read</a>
+                                <a href="" class="text-btn">Bookmark</a>
+                                <a href=""><i class="card-icon fas fa-heart"></i></a>
+                                <a href=""><i class="card-icon fas fa-share-alt"></i></a>
+                                <a href=""><i class="card-icon fas fa-ellipsis-v"></i></a>                      
+                              </div>
```
Comment (shuntzhere) [inline docs/Cards/card.html:215]: Href's

File: docs/Cards/card.html [hunk 8]
```diff
@@ -0,0 +1,328 @@
...
+                         </div>
+                         <div class="card-info">
+                           <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sit, deserunt.</p>
+                         </div>
+                         <div class="card-footer">
+                           <a href="" class="text-btn">Read</a>
+                           <a href="" class="text-btn">Bookmark</a>
+                           <a href=""><i class="card-icon fas fa-heart"></i></a>
+                           <a href=""><i class="card-icon fas fa-share-alt"></i></a>
+                           <a href=""><i class="card-icon fas fa-ellipsis-v"></i></a>
```
Comment (shuntzhere) [inline docs/Cards/card.html:242]: Href's.

File: docs/Cards/card.html [hunk 9]
```diff
@@ -0,0 +1,328 @@
...
+                             <div class="card-info">
+                               <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sit, deserunt.</p>
+                             </div>
+                             <div class="card-footer">
+                               <a href="" class="text-btn">Read</a>
+                               <a href="" class="text-btn">Bookmark</a>
+                               <a href=""><i class="card-icon fas fa-heart"></i></a>
+                               <a href=""><i class="card-icon fas fa-share-alt"></i></a>
+                               <a href=""><i class="card-icon fas fa-ellipsis-v"></i></a>                    
+                             </div>
```
Comment (shuntzhere) [inline docs/Cards/card.html:268]: Href's

File: docs/Cards/card.html [hunk 10]
```diff
@@ -0,0 +1,328 @@
...
+                            </div>
+                       </div>
+                       <div class="example-code">
+                        <pre>
+                          <code class="html hljs xml">
+                            <div class="card ">
+                              <!-- For card with only text remove the card-img-hor div -->
+                              <div class="card-hor">
+                                <div class="card-img-hor">
+                                  <img src="" alt="Card-Image">
```
Comment (shuntzhere) [inline docs/Cards/card.html:278]: Blank src.

File: docs/Cards/card.html [hunk 11]
```diff
@@ -0,0 +1,328 @@
...
+                              <div class="card-info">
+                                <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sit, deserunt.</p>
+                              </div>
+                              <div class="card-footer">
+                                <a href="" class="text-btn">Read</a>
+                                <a href="" class="text-btn">Bookmark</a>
+                                <a href=""><i class="card-icon fas fa-heart"></i></a>
+                                <a href=""><i class="card-icon fas fa-share-alt"></i></a>
+                                <a href=""><i class="card-icon fas fa-ellipsis-v"></i></a>                  
+                              </div>
```
Comment (shuntzhere) [inline docs/Cards/card.html:296]: Href's here.

**Review by shuntzhere — CHANGES_REQUESTED** [review]
Suggested changes to make.

File: CSS/Components/cards.css [hunk 12]
```diff
@@ -0,0 +1,109 @@
+.example-live.card-container{
+    display: flex;
```
Comment (vaibhav-mahalle) [inline CSS/Components/cards.css:2]: use utility classes for recurring code.

File: CSS/Components/cards.css [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+    padding: 0.7rem;
+  }
+  .ecom-card-info {
+    display: flex;
+    justify-content: space-around;
+    font-size: 1.2rem;
+  }
+  .card-footer {
+    font-size: 1.2rem;
+    padding: 0.3rem;
```
Comment (vaibhav-mahalle) [inline CSS/Components/cards.css:39]: same as above

**Review by vaibhav-mahalle — APPROVED** [review]
suggested some minor changes. please look into it.

**Pull request merged**